src/content/contributing/adding-layer-2/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Adding Layer 2s
+description: The policy we use when adding a layer 2 to ethereum.org
+lang: en
+sidebar: true
+---
+
+# Adding layer 2s {#adding-layer-2}
+
+We want to make sure we list the best resources possible so users can navigate the layer 2 space in a safe and confident manner.
+
+Anyone is free to suggest adding a layer 2 on ethereum.org. If there's a layer 2 that we have missed, **[please suggest it](https://github.com/ethereum/ethereum-org-website/issues/new?&template=suggest_layer2.md)!**
+
+We currently list L2s on the following pages:
+
+- [Optimistic rollups](/developers/docs/scaling/optimistic-rollups/)
+- [Zero-knowledge rollups](/developers/docs/scaling/zk-rollups/)
+- [State channels](/developers/docs/scaling/state-channels/)
+<!-- TODO: Add layer-2 page -->
+
+Layer 2 is a relatively new and exciting paradigm for Ethereum. We've tried to create a fair framework for consideration on ethereum.org but the listing criteria will change and evolve over time.
+
+## The decision framework {#decision-framework}
+
+### Criteria for inclusion: the must-haves {#criteria-for-inclusion-the-must-haves}
+
+**[Listing on L2BEAT](https://l2beat.com)**
+
+- In order to be considered, this project must be listed on L2BEAT. L2BEAT provides a robust risk assessment of layer 2 projects that we lean on for evaluating L2 projects. **If the project is not featured on L2BEAT, we will not list them as an L2 on ethereum.org.**
+
+[Learn how to add your L2 project to L2BEAT](https://github.com/l2beat/l2beat/blob/master/CONTRIBUTING.md).
+
+**Open source**
+
+**Layer 2 category**
+
+We currently consider the following to be layer 2 solutions:
+
+- Optimistic rollup
+- Zero-knowledge rollup
+- State channel
+  _We do not consider other scaling solutions to be layer 2._
+
+**Bridges**
+
+- How are users able to onboard to the layer 2?
+
+**Date project went live**
+
+- Newer projects that have not been battle-tested by users are less likely to be listed.
+
+**Wallet support**
+
+- What is the public RPC information for the network? (RPC URL, Chain ID, Currency Symbol, Block Explorer URL)
+- What wallets natively support this layer 2?
```

Comment:
I think this is confusing.

- We can probably remove the first point and keep it in the issue template because any L2 will have a public RPC (Correct me if I'm wrong here)
- The second point is a nice to have

I think we can edit this and move it into the nice to haves

```suggestion
**Native wallet support**

- Do any wallets support the L2 natively?
```